docs/extend/contributing-docs.md

```diff
@@ -2,45 +2,16 @@
 mapped_pages:
   - https://www.elastic.co/guide/en/beats/devguide/current/contributing-docs.html
 applies_to:
-  stack: discontinued 8.18
+  stack: ga 9.0
 ---
 
-# Contributing to the docs [contributing-docs]
-
-The Beats documentation follows the tagging guidelines described in the [Docs HOWTO](https://github.com/elastic/docs/blob/master/README.asciidoc). However it extends these capabilities in a couple ways:
-
-* The documentation makes extensive use of [AsciiDoc conditionals](https://docs.asciidoctor.org/asciidoc/latest/directives/conditionals/) to provide content that is reused across multiple books. This means that there might not be a single source file for each published HTML page. Some files are shared across multiple books, either as complete pages or snippets. For more details, refer to [Where to find the Beats docs source](#where-to-find-files).
-* The documentation includes some files that are generated from YAML source or pieced together from content that lives in `_meta` directories under the code (for example, the module and exported fields documentation). For more details, refer to [Generated docs](#generated-docs).
-
-
-## Where to find the Beats docs source [where-to-find-files]
-
-Because the Beats documentation makes use of shared content, doc generation scripts, and componentization, the source files are located in several places:
-
-| Documentation | Location of source files |
-| --- | --- |
-| Main docs for the Beat, including index files | `<beatname>/docs` |
-| Shared docs and Beats Platform Reference | `libbeat/docs` |
-| Processor docs | `docs` folders under processors in `libbeat/processors/`,`x-pack/<beatname>/processors/`, and `x-pack/libbeat/processors/` |
-| Output docs | `docs` folders under outputs in `libbeat/outputs/` |
-| Module docs | `_meta` folders under modules and datasets in `libbeat/module/`,`<beatname>/module/`, and `x-pack/<beatname>/module/` |
-
-The [conf.yaml](https://github.com/elastic/docs/blob/master/conf.yaml) file in the `docs` repo shows all the resources used to build each book. This file is used to drive the classic docs build and is the source of truth for file locations.
-
-::::{tip}
-If you can’t find the source for a page you want to update, go to the published page at www.elastic.co and click the Edit link to navigate to the source.
-::::
-
-
-The Beats documentation build also has dependencies on the following files in the [docs](https://github.com/elastic/docs) repo:
-
-* `shared/versions/stack/<version>.asciidoc`
-* `shared/attributes.asciidoc`
+# Contributing to the docs
```

Comment:
Do we want to keep the defined anchor?

Reply:
No. Using a defined anchor to link to an h1 heading doesn't work anyway. 🙃  (Pages that still have a defined anchor are just left over from the migration.)

Reply:
TIL. Thank you!